adr/0014-home-assistant-supervised.md

```diff
@@ -0,0 +1,78 @@
+# 0012. Installation method: Home Assistant Container
+
+Date: 2020-06-03
+
+## Status
+
+Accepted
+
+## Context
+
+Define a supported installation method as per [ADR-0012](https://github.com/home-assistant/architecture/blob/master/adr/0012-define-supported-installation-method.md).
+
+## Decision
+
+This installation method provides the full Home Assistant experience on a regular operating system. This means, all components from the Home Assistant method are used, except for the Home Assistant Operating System. We don’t support using the host system for other applications than the Supervisor. You need to understand that the Supervisor is not just an application, it is a full appliance.
+
+By not using the Home Assistant Operating System, the user is responsible for making sure that all required components are installed and maintained. Required components and their versions will change over time. We are not responsible for issues that come up with your host or wrong configuration.
+
+This method is considered advanced and should only be used if one is an expert in managing a Linux operating system, Docker and networking.
+
+### Supported Operating System, System dependencies and versions
+
+Docker CE (Community Edition) is the only supported containerization method for Home Assistant Supervised. We only support FSH 3.0 on the host file system.
```

Comment:
Is this meant to be [FHS](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/index.html)?

Reply:
Yes, that is a typo.